Marcdragon123/storage

marcdragon123 opened the pull request:
All that this pull request need is to be fully tested, have unit tests written and reviewed @NoahCF @Aboumicho @lambhunter0 (only if you feel up to it) If you guys write unit tests you can push to my branch,

marcdragon123 committed:
Implementing Storage for stored values fix#29

marcdragon123 committed:
Implementing Storage for stored values closed#29

Arkondoma committed:
#29 closed

@arkondoma @marcdragon123

Modified variable names for saving values on the calculator app

Arkondoma committed:
#29 closed

@arkondoma @marcdragon123

Fixed listeners missing updated variable names

marcdragon123 committed:
#29 Implementing the delete on exit that should close the temp files when the app is completely closed. @marcdragon123 @arkondoma

ModestosV commented:
Right now if you enter something like 8/2 without entering, you will store the 2. Either force the user to press = and then store the result or store the entire equation,

Commit:
Wrote tests on Store Value, all pass on answers

Arkondoma committed:
@arkondoma

Added unit test for save/load feature, not currently working, needs to access file. Currently returns void value

Arkondoma committed:
Merge remote-tracking branch 'origin/marcdragon123/Storage' into marcdragon123/Storage

# Conflicts:
#	app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt

Arkondoma committed:
@arkondoma

Added unit test for save/load feature, not currently working, needs to access file. Currently returns void value

marcdragon123 committed:
#29 Fixed the tests and made them pass @marcdragon123 @arkondoma @aboumicho

marcdragon123 committed:
#29 Changed functionality to make it more intuitive and changed the tests to function correctly. @marcdragon123 @arkondoma @aboumicho

marcdragon123 committed:
Merge branch 'master' into marcdragon123/Storage

# Conflicts:
#	app/src/main/kotlin/com/simplemobiletools/calculator/helpers/CalculatorImpl.kt

marcdragon123 committed:
Merge branch 'master' into marcdragon123/Storage

# Conflicts:
#	app/src/main/kotlin/com/simplemobiletools/calculator/helpers/CalculatorImpl.kt

marcdragon123 committed:
Merge branch 'master' into marcdragon123/Storage

# Conflicts:
#	app/src/main/kotlin/com/simplemobiletools/calculator/helpers/CalculatorImpl.kt

marcdragon123 committed:
Merge branch 'master' into marcdragon123/Storage

marcdragon123 committed:
#39 Fixed the error message to become a popup. @marcdragon123

marcdragon123 committed:
Merge branch 'marcdragon123/Storage' of https://github.com/ModestosV/Simple-Calculator-DreamTeam into marcdragon123/Storage

# Conflicts:
#	app/src/main/kotlin/com/simplemobiletools/calculator/helpers/CalculatorImpl.kt

marcdragon123 committed:
#39 Added a popup for the error message and fixed imports. @marcdragon123

ModestosV reviewed (approved):
Coding style matches our coding standards. Feature works as intended and handles cases where there is no value to write/read. This pull request completely handles issue #29 .
